src/wp-includes/compat.php

```diff
@@ -549,6 +549,102 @@ function str_ends_with( $haystack, $needle ) {
 	}
 }
 
+if ( ! function_exists( 'array_find' ) ) {
+	/**
+	 * Polyfill for `array_find()` function added in PHP 8.4.
+	 *
+	 * Searches an array for the first element that passes a given callback.
+	 *
+	 * @since 6.8.0
+	 *
+	 * @param array $array The array to search.
+	 * @param callable $callback The callback to run for each element.
+	 *
+	 * @return mixed|null The first element in the array that passes the `$callback`, otherwise null.
```

Comment:
```suggestion
	 * @param callable $callback The callback to run for each element.
	 * @return mixed|null The first element in the array that passes the `$callback`, otherwise null.
```